extensions/ima/src/main/java/com/google/android/exoplayer2/ext/ima/ImaAdsLoader.java

```diff
@@ -135,6 +138,18 @@ public Builder setAdEventListener(AdEventListener adEventListener) {
       return this;
     }
 
+    /**
+     * Sets the ad UI elements to be rendered by the IMA SDK.
+     *
+     * @param adUiElements The ad UI elements to be rendered by the IMA SDK.
+     * @return This builder, for convenience.
+     * @see AdsRenderingSettings#setUiElements(Set)
+     */
+    public Builder setAdUiElements(Set<UiElement> adUiElements) {
+      this.adUiElements = Assertions.checkNotNull(adUiElements);
```

Comment:
It might be worth storing a copy of the set, in case the application modifies it.